packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3540,6 +3541,7 @@ export default class Meeting extends StatelessWebexPlugin {
    */
   public setPermissionTokenPayload(permissionToken: string) {
     this.permissionTokenPayload = jwt.decode(permissionToken);
+    this.tokenReceivedTime = new Date().getTime();
```

Comment:
it would be good to group the `permissionTokenPayload` and `tokenReceivedTime` under some common property in the Meeting object like "permissionTokenInfo" or something like that, otherwise we end up with a lot of scattered properties in the Meeting object, especially that the name "tokenReceivedTime" doesn't say which token it is about

Reply:
`tokenReceivedTime` can be renamed to `permissionTokenReceivedLocalTime`, if I remove `permissionTokenPalyload` from the meeting object and move it into another property say 'permissionTokenInfo' as suggested then that will have lot more code/unittest changes. Or we can have `permissionTokenPalyload` in 2 places, directly on meetingObject and also in the new property

Reply:
> then that will have lot more code/unittest changes

but it's just a simple renaming everywhere from `.permissionTokenPayload` to `.permissionTokenInfo.payload`. Anyway, just renaming it to `permissionTokenReceivedLocalTime` is I guess fine too